### PR TITLE
onert(nnfw) uses 'cpu;bcq' as default backend

### PR DESCRIPTION
**Pull request by chunseoklee**
For now, there is no api to use bcq backend from android java
application. Thus, as a workaround, set 'cpu;bcq' as a deafult backend.

Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

---
# [Template] PR Description

In general, github system automatically copies your commit message for your convenience.
Please remove unused part of the template after writing your own PR description with this template.
```bash
$ git commit -s filename1 filename2 ... [enter]

Summarize changes in around 50 characters or less

More detailed explanatory text, if necessary. Wrap it to about 72
characters or so. In some contexts, the first line is treated as the
subject of the commit and the rest of the text as the body. The
blank line separating the summary from the body is critical;
various tools like `log`, `shortlog` and `rebase` can get confused 
if you run the two together.

Further paragraphs come after blank lines.

**Changes proposed in this PR:**
- Bullet points are okay, too
- Typically a hyphen or asterisk is used for the bullet, preceded
  by a single space, with blank lines in between, but conventions vary here.

Resolves: #123
See also: #456, #789

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

**How to evaluate:**
1. Describe how to evaluate in order to be reproduced by reviewer(s).

Add signed-off message automatically by running **$git commit -s ...** command.

$ git push origin <your_branch_name>
```

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #include <nnfw.h>
 
 /** backends supported by nnfw */
-#define NNFW_CPU_BACKEND  "cpu"
+#define NNFW_CPU_BACKEND  "cpu;bcq"
```
Comment (kparichay) [inline ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:40]: This does change the default backend but I recommend to create a NNFW_CPU_BCQ_BACKEND here and use it.
Further, register a ACCL_CPU_BCQ_STR and ACCL_CPU_BCQ at `gst/nnstreamer/include/nnstreamer_plugin_api_filter.h` and then update the auto/default backend at line 55 to 60 for various architectures.

**Comment by jaeyun-jung** [comment]
Discussion

@chunseoklee 
IMO, this is ok if `cpu;bcq` on NNFW determines backend,
1. if nnfw-boost lib exists, run on boost
2. if nnfw-boost is not available, run on cpu

Also, please check the patch https://github.com/nnstreamer/nnstreamer/pull/2687
This PR is to pass option string to NNFW. (e.g. custom str = backend:cpu;bcq)

**Review by jaeyun-jung — APPROVED** [review]

File: ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c (hunk 1 above)
Comment (myungjoo) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:40]: Until we get it updated, let's use either:
```
#define NNFW_CPU_BACKEND  "cpu;bcq"
```
or
```
#define NNFW_DEFAULT_BACKEND NNFW_CPU_BACKEND ";bcq"
```
Let's not touch the API related headers at this moment (release imminent)

**Review by myungjoo — APPROVED** [review]
LGTM

File: ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c (hunk 1 above)
Comment (myungjoo) [reply ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c:40]: Besides, as long as bcq enabled just yet another algorithm using CPU instructions, having "cpu;bcq" as CPU_BACKEND  isn't terribly wrong.

**Comment by leemgs** [comment]
[Note]  Dongsoo Lee has made the kernel (BCQ) to run a BIQGEMM-based network model. "BCQ" means Bitwise Compressed Quantization.

**Pull request merged**